docs/velocity/dev/how-to/plugin-messaging.md

```diff
@@ -0,0 +1,72 @@
+---
+slug: /dev/plugin-messaging
+---
+
+# Plugin Messaging
+
+First introduced in [2012](https://web.archive.org/web/20220711204310/https://dinnerbone.com/blog/2012/01/13/minecraft-plugin-channels-messaging/),
```

Comment:
Do we want to mention somewhere that it is bungeecord:main on the proxy side but BungeeCord on the server side ?